web/src/components/njwds-extended/HorizontalStepper.tsx

```diff
@@ -1,12 +1,12 @@
-import { MediaQueries } from "@/lib/PageSizes";
 import { StepperStep } from "@/lib/types/types";
```

Comment:
I don't love how both HorizontalStepper and BusinessFormationPaginator, share controll of the refocus responsibility (the former for focusing on the stepper and the latter for focusing on the errors above the components). 

Lead to some awkwardness where the HorizontalStepper has to kinda be aware of whether or not errors exist via props and then suppress its own refocus behavior. 

I didn't see a great / easy way to refactor this. If anyone has a good idea here lmk. Or if naming could be clearer lmk.